.github/workflows/cypress.yaml

```
@@ -5,6 +5,14 @@ on:
     branches:
       - develop
       - staging
+    paths:
+      - 'src/**'
+      - 'cypress/**'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'plugins/**'  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)